pymodbus/repl/client/mclient.py

```diff
@@ -71,14 +71,13 @@ class ExtendedRequestSupport:  # pylint: disable=(too-many-public-methods
 
     @staticmethod
     def _process_exception(resp, **kwargs):
-        """ Internal process exception. """
-        unit = kwargs.get("unit")
-        if unit == 0:
+        """Set internal process exception."""
+        if not kwargs.get("unit"):
```

Comment:
Broadcast to be assumed only when the unit is 0, this will assume a broadcast request when `unit` is not available in the `kwargs` and prints a wrong message instead of the actual exception.

Reply:
the problem is that unit is an unused variable so I tried to get around that.